test/middleware/authentication.middleware.unit.ts

```diff
@@ -29,4 +30,12 @@ describe("authentication middleware tests", () => {
     expect(mockAuthMiddleware).toHaveBeenCalledWith(expectedAuthMiddlewareConfig);
     expect(mockAuthReturnedFunction).toHaveBeenCalledWith(req, res, next);
   });
+
+  it("should not call authentication middleware when loading accessibility page", () => {
```

Comment:
Maybe call this test "should not redirect to sign in when url is on whitelist"

Reply:
Amended